src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -312,6 +319,21 @@ boolean isFlowBasedCallingMode() {
         return flowMode != FlowMode.NONE;
     }
 
+    // Default constructor
+    public HaplotypeCallerArgumentCollection() {
+        super();
+    }
+
+    // Private constructor for use in copyWithNewPloidy
+    // Allows you to make new instances of hcArgs with synced fields except for genotyper ploidy
+    private HaplotypeCallerArgumentCollection(int ploidy) {
+        this.standardArgs.genotypeArgs.samplePloidy = ploidy;
```

Comment:
This works? This looks terrifying...

Reply:
I swear I found this idea somewhere online and I can't find where anymore... Do you have any suggestions on doing this in a better way?

Reply:
Given that both of us are weary of this option... if you keep this confusing copy constructor (that i'm really not sure works at all this is horrifying...) you are going to need to check in some unit tests where you call this some times on a on-default argument collection (probably also non-default sub-argument collections) and demonstrate that everything is identical after calling this except for ployidy.

Reply:
the whole point was so they would be in-sync if something changes in one? that is easily demonstrated with a test...